packages/manager/modules/support/src/tickets/new-ticket/issues-form/issues-form.html

```diff
@@ -80,17 +80,25 @@
           </li>
         </ul>
       </oui-tile>
-      <p class="mt-4"
-         data-translate="ovhManagerSupport_new_helpfull">
-      </p>
-      <oui-button data-variant="secondary"
-                  data-text="{{ 'ovhManagerSupport_new_helpfull_no' | translate }}"
-                  data-on-click="$ctrl.submitForm(false)">
-      </oui-button>
-      <oui-button data-variant="primary"
-                  data-text="{{ 'ovhManagerSupport_new_helpfull_yes' | translate }}"
-                  data-on-click="$ctrl.submitForm(true)">
-      </oui-button>
+      <div data-ng-if="$ctrl.guides.length || $ctrl.tips.length">
+        <p class="mt-4"
+           data-translate="ovhManagerSupport_new_helpfull">
+        </p>
+        <oui-button data-variant="secondary"
+                    data-text="{{:: 'ovhManagerSupport_new_helpfull_no' | translate }}"
```

Comment:
Don't use `text`, use transpilation instead:
````js
<oui-button
  data-variant="secondary"
  data-on-click="$ctrl.submitForm(false)">
  <span data-translate="ovhManagerSupport_new_helpfull_no"></span>
</oui-button>
````

Also, looks like this submits a form, then the button `type` should be `submit`;)